docs/ad/api.md

```diff
@@ -149,39 +149,79 @@ Passes a date range to the anomaly detector to return any anomalies within that
 ```json
 POST _opendistro/_anomaly_detection/detectors/<detector_id>/_preview
 {
-    "period_start":1573089773852,
-    "period_end":1573521773853
+  "period_start": 1573089773852,
+  "period_end": 1573521773853
 }
 ```
 
 #### Sample response
 
 ```json
 {
-    "anomaly_result":[
-        ...
+  "anomaly_result": [
+    {
+      "detector_id": "B2aJlG4B04EFYPipCRO4",
+      "anomaly_grade": 0.1,
+      "confidence": 0.7,
+      "feature_data": [
         {
-            "detector_id":"B2aJlG4B04EFYPipCRO4",
-            "anomaly_grade":0.1,
-            "confidence":0.7,
-            "feature_data":[
-                {
-                    "feature_id":"value_sum",
-                    "feature_name":"Value Sum",
-                    "data":20
-                }
-            ],
-            "start_time":1573514220000,
-            "end_time":1573514520000
-        },
-        ...
-    ],
-    "anomaly_detector":{
-        "name":"test-detector",
-        "description":"test detector",
-        "time_field":"timestamp",
-        ...
+          "feature_id": "value_sum",
+          "feature_name": "Value Sum",
+          "data": 20
+        }
+      ],
+      "start_time": 1573514220000,
+      "end_time": 1573514520000
```

Comment:
`start_time` -> `data_start_time`
`end_time` -> `data_end_time`
And , add two more time fields: `execution_start_time` and `execution_end_time`